mantis-publish/mantis-publish-core/src/main/java/io/mantisrx/publish/internal/mql/MQLSubscription.java

```diff
@@ -117,8 +118,26 @@ public boolean matches(Event event) {
         }
     }
 
+	private List<String> sanitizeStreamSubjects(final List<String> subjects) {
+        return subjects.stream()
+                .map((String s) -> {
+                    if (s.toLowerCase().equals("observable") ||
+                            s.toLowerCase().equals("stream") ||
+                            s.toLowerCase().equals("defaultstream")) {
```

Comment:
nit: Could we introduce a new Stream DS (or a Subject DS, not sure what to call it) that has properties around whether it's a default stream or not for instance? We can also expose that through the contract for the Subscription class instead of just returning a list of strings.